RawWaterManagement/OpenChannelCurve/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,100 @@
+{
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -72.868304,
+        135.670598
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Finish interview own industry else yet road. Report authority property student quite understand and. Sing world inside recently wife space some possible.",
+      "addressLocality": "Least white another only manager kind better. Hair doctor worker act employee week step. Wait finally money continue customer maintain including ability. Else big science director Republican.",
+      "addressRegion": "Section decision road apply three hold. Teacher yes election personal sell very.",
+      "addressCountry": "Its drive well five. Need yet box stand manager. Manage them right.",
+      "postalCode": "Effect party finish entire a agree. Hope process assume. Writer trip stuff else commercial few.",
+      "postOfficeBoxNumber": "Government report big those lawyer bank. News court matter food remember major.",
+      "areaServed": "There knowledge finish front dinner movie. Citizen why fine travel."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Tonight happen which consumer one. Guess actually if money key. Hospital interesting can hair level. Remember hand agent."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:OHYG:26400083",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2009-05-25T07:23:18Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1970-11-27T12:47:48Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Hospital go instead action always phone pattern month. Teach tell group season eight amount should. Available parent maybe occur."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Through everybody face appear big. Cell none put four war gas. Serve will thousand may peace. Country cause watch from."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Bank necessary sister theory imagine mission. Professor card how physical guess. Free meet him."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Exist allow purpose grow serious off six. Best staff experience pick thing order four free. Fear because major rock nearly reduce."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Production person authority method."
+  },
+  "owner": {
```

Comment:
type suggested to be Relationship
note:

    Relationships should not be modelled as properties, otherwise you completely lost the interest of NGSI-LD which is based on graph.
    Arrays in NGSI-LD are stored as strings so you can not use the API to query indiviudual alements in a array.
    Relationships can not be provided within arrays.

When you need to have several relationships with the same name, then you have to use the datasetId

Example for a relationship where one entity would "monitor" 2 different rooms. It thus have 2 relationshipsnames "monitored" directed toward room01 and room02. The datasetId allows to differentiate the 2 relationships.
"monitors":[
{
"type":"Relationship",
"object":"urn:ngsi-ld:Room:01",
"datasetId":"urn:ngsi-ld:dataSet:01"},
},
"type":"Relationship",
"object":"urn:ngsi-ld:Room:02" ,
"datasetId":"urn:ngsi-ld:dataSet:02"}
]